src/lib/kw_config_loader.sh

```diff
@@ -330,7 +330,14 @@ function load_configuration()
   target_array_local="${target_array}_local"
 
   target_config_file="${target_config}.config"
-  parse_configuration "${KW_ETC_DIR}/${target_config_file}" "$target_array" "$target_array_global"
+
+  # Check if the target configuration file already exists in the expected locations
+  if [[ -f "${KW_ETC_DIR}/${target_config_file}" || -f "${XDG_CONFIG_HOME:-"${HOME}/.config"}/${KWORKFLOW}/${target_config_file}" || -f "${PWD}/${KW_DIR}/${target_config_file}" ]]; then
+    echo "Configuration file ${target_config_file} already exists. Skipping creation to avoid duplication."
```

Comment:
We avoid the use of `echo` in the kw project. Check:
https://kworkflow.org/content/codingstyle.html#use-printf-instead-of-echo